backend/src/main/resources/data.sql

```diff
@@ -0,0 +1,106 @@
+    (23, 'JNW12300KL8945029', 'WWGH00', 'Nissan', 'Navara', 2010, 155000, 'Some description', 8),
+    (24, 'VF1Z123KRH408UH33', 'WS90WQ', 'Renault', 'Scenic', 2009, 210000, 'Some description', 5),
+    (25, 'WP0122347650FH44F', 'KR22RT', 'Porsche', 'Carrera', 2006, 269350, 'Some description', 6);
+
+INSERT INTO customers(id, name, surname, phone, address) VALUES
+    (1, 'Jan', 'Kowalski', '665456987', 'Kaliska 12, Bełchatów'),
+    (2, 'Maciej', 'Zalewski', '126434567', 'Warszawska 15, Kraków'),
+    (3, 'Aneta', 'Korczyńska', '789456123', 'Miodowa 23, Nysa'),
+    (4, 'Rafał', 'Nowak', '654321987', 'Piłsudskiego 20, Warszawa'),
+    (5, 'Marta', 'Lewicka', '660369852', 'Antracytowa 56, Kraków'),
+    (6, 'Bartosz', 'Lewicki', '721435678', 'Antracytowa 56, Kraków'),
+    (7, 'Kamil', 'Kwiatkowski', '620908901', 'Wiśniowa 10, Siedlce'),
+    (8, 'Kamila', 'Baran', '567890121', 'Dębowa 10, Częstochowa'),
+    (9, 'Magda', 'Kwiatkowska', '500100890', 'Wojska Polskiego 19, Warszawa'),
+    (10, 'Jan', 'Kowalski', '882555444', 'Fabryczna 10, Piotrków Trybunalski'),
+    (11, 'Jan', 'Nowak', '666555444', 'Kolorowa 79, Łódź'),
+    (12, 'Mariusz', 'Drągowski', '789456000', 'Leśna 60, Wrocław'),
+    (13, 'Anna', 'Wolska', '802134678', 'Sienkiewicza 2, Bełchatów'),
+    (14, 'Aneta', 'Nowak', '620908900', 'Mickiewicza 50, Katowice'),
+    (15, 'Adrian', 'Stasiak', '777888999', 'Zielona 33, Janów Lubelski'),
+    (16, 'Agnieszka', 'Stasiak', '745966365', 'Topolowa 20, Rzeszów'),
+    (17, 'Marcin', 'Wolski', '555444333', 'Lipińskiego 45, Poznań'),
+    (18, 'Ireneusz', 'Krzemiński', '888777666', 'Cebulowa 7, Wejherowo'),
+    (19, 'Beata', 'Pawlak', '789454987', 'Boczna 120, Katowice'),
+    (20, 'Jerzy', 'Mazur', '521321457', 'Zamknięta 1, Suwałki'),
+    (21, 'Kacper', 'Lewandowski', '504264111', 'Złota 21, Legnica'),
+    (22, 'Bartosz', 'Izdebski', '741852963', 'Pomorska 17, Gdynia'),
+    (23, 'Adam', 'Grzanka', '600400900', 'Srebrna 40, Warszawa'),
+    (24, 'Anna', 'Kolanko', '852741963', 'Srebrna 40, Warszawa'),
+    (25, 'Jacek', 'Górnik', '636696000', 'Obozowa 25, Wrocław');
+
+INSERT INTO customers_cars(customers_id, car_id) VALUES
+    (1, 2),
+    (2, 1),
+    (3, 14),
+    (4, 3),
+    (4, 15),
+    (5, 5),
+    (6, 5),
+    (7, 22),
+    (7, 23),
+    (7, 24),
+    (8, 18),
+    (9, 21),
+    (10, 12),
+    (11, 4),
+    (12, 8),
+    (13, 25),
+    (14, 17),
+    (14, 6),
+    (15, 9),
+    (15, 19),
+    (16, 9),
+    (17, 20),
+    (18, 7),
+    (19, 16),
+    (20, 10),
+    (21, 11),
+    (22, 13),
+    (23, 25),
+    (24, 25),
+    (25, 8);
+
+INSERT INTO roles(id,name) VALUES
+    (1, 'admin');
+    (2, 'user'),
+
+INSERT INTO accounts(id, login, password, email, login_attempts, registration_date, last_successful_login,
+                     last_failed_login, role_id) VALUES
+    (1, 'admin', 'admin', 'admin@mail.com', 6, 2021-05-24 16:00:00.000, 2021-05-26 16:00:00.000, 2021-05-25 16:00:00.000, 1),
```

Comment:
Password should be stored encrypted

Reply:
You can use this -> https://bcrypt-generator.com/

Reply:
Done